vendor/engines/bulk_email/app/services/bulk_email/content_generator.rb

```diff
@@ -25,17 +27,21 @@ def wrap_text(text, recipient_name = nil)
 
     def greeting(recipient_name = nil)
       [
-        I18n.t("bulk_email.body.greeting", recipient_name: recipient_name || DEFAULT_RECIPIENT_NAME),
+        text("body.greeting", recipient_name: recipient_name || DEFAULT_RECIPIENT_NAME),
         reason_statement,
       ].compact.join("\n\n")
     end
 
     def signoff
       if facility.single_facility?
-        I18n.t("bulk_email.body.signoff", facility_name: facility.name)
+        text("body.signoff", facility_name: facility.name)
       end
     end
 
+    def translation_scope
```

Comment:
And the reason I switched to using the `text` helper over regular `I18n` was to get the `!app_name!` interpolation feature.